portability-spi-cloud/src/main/java/org/dataportabilityproject/spi/cloud/types/PortabilityJob.java

```diff
@@ -59,7 +59,7 @@
 
   public static PortabilityJob.Builder builder() {
     LocalDateTime now = LocalDateTime.now();
-    return new AutoValue_PortabilityJob.Builder()
+    return new org.dataportabilityproject.spi.cloud.types.AutoValue_PortabilityJob.Builder()
```

Comment:
Can we add a note as to why it has to be fully qualified? Does it have to happen for JobAuthorization as well?

Reply:
Yes, good call. IntelliJ built fine without it, but when you opened the class it would show red underlined AutoValue_ class. Now it's fixed. Added comments as well.